src/modules/preview-site/components/preview-site-row.tsx

```diff
@@ -122,13 +132,23 @@ export function PreviewSiteRow( {
 					</div>
 					<div className="w-[100px] text-[#757575] flex items-center pl-4">{ countDown }</div>
 					<div className="w-[60px] flex justify-end">
-						<PreviewActionButtonsMenu
-							snapshot={ snapshot }
-							selectedSite={ selectedSite }
-							disabledUpdate={ disabledUpdate }
-							updateButtonTooltipContent={ updateButtonTooltipContent }
-							showUpdateTooltip={ showUpdateTooltip }
-						/>
+						{ isExpired ? (
+							<Button
+								variant="link"
+								onClick={ () => removeSnapshot( snapshot ) }
+								className={ '!text-a8c-gray-70 hover:!text-a8c-red-50' }
```

Comment:
This is the same styling as we use for the `Disconnect` button on the Sync tab:

![Markup on 2025-02-10 at 16:14:53](https://github.com/user-attachments/assets/002db1ed-f8c7-4048-a9e9-62390a6d6772)

(including red color on hover)

Reply:
At the end I changed the color of the `Clear` button to `text-a8c-blueberry` (https://github.com/Automattic/studio/pull/913/commits/83a21d094e33915752e35eea550bcacf408b7a22) - to match it with the design in Figma.

I have also kept the red color on hover.